src/sagemaker_containers/server.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import signal
+import subprocess
+import sys
+
+import sagemaker_containers
+
+NGINX_PID = 0
+
+
+def add_terminate_signal(process):
+    def terminate(signal_number, stack_frame):
+        process.terminate()
+
+    signal.signal(signal.SIGTERM, terminate)
+
+
+def start_server(module_app):
+
+    env = sagemaker_containers.environment.ServingEnvironment()
+    gunicorn_bind_address = '0.0.0.0:8080'
+
+    nginx = None
+
+    if env.use_nginx:
+        gunicorn_bind_address = 'unix:/tmp/gunicorn.sock'
+        nginx = subprocess.Popen(['nginx', '-c', '/tmp/nginx.conf'])
```

Comment:
Do we need access and error logs: https://github.com/aws/sagemaker-containers/blob/master/src/container_support/serving.py#L87